### PR TITLE
Recipe: add completionTime attribute to recipe class

### LINKED ISSUE #41: Add completion time attribute to Recipe class
Add completion time attribute to recipe class in order to store/display total cooking time of each recipe

### PR DESCRIPTION
**Pull request by aidanaden**
Fixes #41

### CODE REVIEW

File: src/main/java/seedu/address/model/recipe/Recipe.java [hunk 1]
```diff
@@ -120,7 +128,8 @@ public String toString() {
         // TODO: edit if needed
         StringBuilder sb = new StringBuilder();
         sb.append(getName())
-                .append(String.format("; Portions: %2f", getPortion()));
+                .append(String.format("; Completion time: %d, Portions: %2f",
+                        getCompletionTime(), getPortion()));
```
Comment (Oddcorner) [inline src/main/java/seedu/address/model/recipe/Recipe.java:132]: How long is a unit of completion time? I think it should be reflected here.
```suggestion
                .append(String.format("; Completion time: %d (mins), Portions: %2f",
                        getCompletionTime(), getPortion()));
```

Comment (aidanaden) [reply src/main/java/seedu/address/model/recipe/Recipe.java:132]: Yep seems gud LFG

**Review by yusufaine — COMMENTED** [review]
LGTM 🌮

**Review by Oddcorner — APPROVED** [review]
LGTM! 🌮

File: src/main/java/seedu/address/model/recipe/CompletionTime.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        value = completionTime;
+    }
+
+    public static boolean isValidCompletionTime(int test) {
+        return (test > 0);
+    }
+
+    @Override
+    public String toString() {
+        return String.format("%d", value);
```
Comment (yusufaine) [inline src/main/java/seedu/address/model/recipe/CompletionTime.java:36]: minor suggestion (doesnt affect functionality):
```suggestion
        return String.format("%d mins", value);
```

File: src/main/java/seedu/address/model/recipe/Recipe.java [hunk 3]
```diff
@@ -120,7 +128,8 @@ public String toString() {
         // TODO: edit if needed
         StringBuilder sb = new StringBuilder();
         sb.append(getName())
-                .append(String.format("; Portions: %2f", getPortion()));
+                .append(String.format("; Completion time: %d (mins), Portions: %2f",
+                        getCompletionTime().value, getPortion()));
```
Comment (yusufaine) [inline src/main/java/seedu/address/model/recipe/Recipe.java:132]: minor suggestion follow-up:

```suggestion
                .append(String.format("; Completion time: %s, Portions: %2f",
                        getCompletionTime(), getPortion()));
```

**Review by yusufaine — APPROVED** [review]
LGTM 🌮 

Only left some minor suggestion should you choose to commit them

**Review by gigituang — APPROVED** [review]

**Pull request merged**